tritonserver/src/main/java/org/bytedeco/tritonserver/presets/tritonserver.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2018-2021 Samuel Audet
+ *
+ * Licensed either under the Apache License, Version 2.0, or (at your option)
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation (subject to the "Classpath" exception),
+ * either version 2, or any later version (collectively, the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.gnu.org/licenses/
+ *     http://www.gnu.org/software/classpath/license.html
+ *
+ * or as provided in the LICENSE.txt file that accompanied this code.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bytedeco.tritonserver.presets;
+
+import java.util.List;
+import org.bytedeco.javacpp.ClassProperties;
+import org.bytedeco.javacpp.LoadEnabled;
+import org.bytedeco.javacpp.Loader;
+import org.bytedeco.javacpp.annotation.Platform;
+import org.bytedeco.javacpp.annotation.Properties;
+import org.bytedeco.javacpp.tools.Info;
+import org.bytedeco.javacpp.tools.InfoMap;
+import org.bytedeco.javacpp.tools.InfoMapper;
+
+import org.bytedeco.cuda.presets.cudart;
+import org.bytedeco.cuda.presets.cublas;
+import org.bytedeco.cuda.presets.cudnn;
+import org.bytedeco.cuda.presets.nvrtc;
+import org.bytedeco.tensorrt.presets.nvinfer;
+import org.bytedeco.tensorrt.presets.nvinfer_plugin;
+import org.bytedeco.tensorrt.presets.nvonnxparser;
+import org.bytedeco.tensorrt.presets.nvparsers;
+
+/**
+ *
+ * @author Jack He
+ */
+@Properties(
+    inherit = {cublas.class, cudnn.class, nvrtc.class, nvinfer.class, nvinfer_plugin.class, nvonnxparser.class, nvparsers.class},
+    value = {
+        @Platform(
+            value = {"linux-arm64", "linux-ppc64le", "linux-x86_64", "windows-x86_64"},
+            compiler = "cpp11",
+            include = {"tritonbackend.h", "tritonrepoagent.h", "tritonserver.h"},
```

Comment:
Here, "tritonserver.h" should probably be moved before "tritonbackend.h".

Reply:
> Here, "tritonserver.h" should probably be moved before "tritonbackend.h".
Yes, it works! It gets compiled eventually. Also, I removed the nouse gererated TensorRT files.

Reply:
Right, simply remove src/gen and rerun the build to get rid of old files.